middle_end/flambda2/types/env/typing_env.ml

```diff
@@ -599,6 +599,17 @@ let mem_simple ?min_name_mode t simple =
     ~name:(fun name ~coercion:_ -> mem ?min_name_mode t name)
     ~const:(fun _ -> true)
 
+let alias_is_bound_before t ~bound_name ~alias =
```

Comment:
From a french point of view on before, it is not obvious that this is strictly before. Also I'm not sure but should it be 'earlier' rather than 'before' ?
So either `alias_is_bound_strictly_before` or `binding_time_of_alias_is_strictly_earlier`

Reply:
Renamed to `alias_is_bound_strictly_earlier`